lib/jsdom/living/nodes/HTMLTableElement-impl.js

```diff
@@ -62,33 +114,25 @@ class HTMLTableElementImpl extends HTMLElementImpl {
   createTHead() {
     let el = this.tHead;
     if (!el) {
-      el = this._ownerDocument.createElement("THEAD");
-      this.appendChild(el);
+      el = this.tHead = this._ownerDocument.createElement("THEAD");
```

Comment:
This and other changes move away from a more literal implementation of the spec... any particular reason to make them?

Reply:
creatTHead seems to me to have exactly the same insertion rules (before first non-colgroup, non-caption element), so I think this should actually make it more compliant?

Reply:
Hmm, I must have been looking at createTFoot(), weird. I guess it's nicer this way then.